packages/buildbucket-dart/lib/src/generated/go.chromium.org/luci/buildbucket/proto/build.pbenum.dart

```diff
@@ -14,20 +14,14 @@ import 'dart:core' as $core;
 import 'package:protobuf/protobuf.dart' as $pb;
 
 class BuildInfra_Buildbucket_ExperimentReason extends $pb.ProtobufEnum {
-  static const BuildInfra_Buildbucket_ExperimentReason EXPERIMENT_REASON_UNSET =
-      BuildInfra_Buildbucket_ExperimentReason._(0, _omitEnumNames ? '' : 'EXPERIMENT_REASON_UNSET');
-  static const BuildInfra_Buildbucket_ExperimentReason EXPERIMENT_REASON_GLOBAL_DEFAULT =
-      BuildInfra_Buildbucket_ExperimentReason._(1, _omitEnumNames ? '' : 'EXPERIMENT_REASON_GLOBAL_DEFAULT');
-  static const BuildInfra_Buildbucket_ExperimentReason EXPERIMENT_REASON_BUILDER_CONFIG =
-      BuildInfra_Buildbucket_ExperimentReason._(2, _omitEnumNames ? '' : 'EXPERIMENT_REASON_BUILDER_CONFIG');
-  static const BuildInfra_Buildbucket_ExperimentReason EXPERIMENT_REASON_GLOBAL_MINIMUM =
-      BuildInfra_Buildbucket_ExperimentReason._(3, _omitEnumNames ? '' : 'EXPERIMENT_REASON_GLOBAL_MINIMUM');
-  static const BuildInfra_Buildbucket_ExperimentReason EXPERIMENT_REASON_REQUESTED =
-      BuildInfra_Buildbucket_ExperimentReason._(4, _omitEnumNames ? '' : 'EXPERIMENT_REASON_REQUESTED');
-  static const BuildInfra_Buildbucket_ExperimentReason EXPERIMENT_REASON_GLOBAL_INACTIVE =
-      BuildInfra_Buildbucket_ExperimentReason._(5, _omitEnumNames ? '' : 'EXPERIMENT_REASON_GLOBAL_INACTIVE');
-
-  static const $core.List<BuildInfra_Buildbucket_ExperimentReason> values = <BuildInfra_Buildbucket_ExperimentReason>[
+  static const BuildInfra_Buildbucket_ExperimentReason EXPERIMENT_REASON_UNSET = BuildInfra_Buildbucket_ExperimentReason._(0, _omitEnumNames ? '' : 'EXPERIMENT_REASON_UNSET');
```

Comment:
Does the line limit match style?

Reply:
Good catch. I forgot to run the formatter but looks like we ignore these files when linting?